src/pages/login/Login.tsx

```diff
@@ -35,8 +35,8 @@ const Login: FunctionComponent = () => {
     setMaskPassword(!maskPassword);
   };
 
-  const handleLogin = ({ email, password }: FormikValues) => {
-    dispatch(sessionApi.endpoints.login.initiate({ email, password }));
+  const handleLogin = async ({ email, password }: FormikValues) => {
+    await dispatch(sessionApi.endpoints.login.initiate({ email, password }));
```

Comment:
Made this async/await so that Formik knows when to turn `isSubmitting` to true/false and disable the login button to prevent it from being executed multiple times while the call is being made.